docs/src/components/ComponentDoc/ComponentDoc.tsx

```diff
@@ -63,30 +62,95 @@ class ComponentDoc extends React.Component<any, any> {
   }
 
   render() {
+    const getA11ySelectionMessage = {
+      onAdd: item => `${item} has been selected.`,
+      onRemove: item => `${item} has been removed.`,
+    }
+
+    const getA11yStatusMessage = ({
+      isOpen,
+      itemToString,
+      previousResultCount,
+      resultCount,
+      selectedItem,
+    }) => {
+      if (!isOpen) {
+        return selectedItem ? itemToString(selectedItem) : ''
+      }
+      if (!resultCount) {
+        return 'No results are available.'
+      }
+      if (resultCount !== previousResultCount) {
+        return `${resultCount} result${
+          resultCount === 1 ? ' is' : 's are'
+        } available, use up and down arrow keys to navigate. Press Enter key to select.`
+      }
+      return ''
+    }
+
     const { info } = this.props
-    const { activePath, examplesRef } = this.state
+    const { activePath /*, examplesRef*/ } = this.state
 
     return (
-      <DocumentTitle title={`${info.displayName} | Stardust`}>
-        <Grid>
-          <Grid.Row style={topRowStyle}>
-            <Grid.Column>
-              <Header as="h1" content={info.displayName} variables={{ color: 'black' }} />
-              <p>{_.join(info.docblock.description, ' ')}</p>
-              <ComponentAccessibility info={info} />
-              <ComponentDocSee displayName={info.displayName} />
-              <ComponentDocLinks
-                displayName={info.displayName}
-                parentDisplayName={info.parentDisplayName}
-                repoPath={info.repoPath}
-                type={info.type}
-              />
-              <ComponentProps displayName={info.displayName} props={info.props} />
-            </Grid.Column>
-          </Grid.Row>
+      <Provider
+        theme={{
+          componentStyles: {
+            Box: {
+              root: {
+                display: 'flex',
+                flexDirection: 'column',
+                justifyContent: 'space-between',
+              },
```

Comment:
This guy here is the issue with the ChatMessages... The main problem is that, we are merging the docs theme together with teams theme, and this is used in the examples. Any changes we make for the docs site, are applied to the examples as well... @miroslavstastny I know that at some point we were talking about adding reset prop on the Provider, but nothing turn out of that. Do you have any proposal how we can overcome this easily at this moment?

Reply:
We agreed with @alinais to try and get rid of the nested Providers, or introduce different components with custom displayNames, so that we won't affect the examples' theme.

Reply:
@mnajdova, frankly, not sure about that - as, ideally, the point we would like to get to is where there would be Provider-based styles applied to example components. At the same time, we would like to have docs being styled the same way, with the same components' names used.

Could you, please, suggest why we haven't decided just to introduce this flag that would clean Provider's theme context, the one you were talking about? It seems like the most clean solution, and the one that will get us closer we would like to be at. If next day we will decide to use iframe-based approach, or anything else - with that all the markup and styling code both of docs, as well as of docs examples, everything would remain the same with no changes being necessary.